core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -150,6 +158,9 @@ private[spark] class ExecutorAllocationManager(
   // Metric source for ExecutorAllocationManager to expose internal status to MetricsSystem.
   val executorAllocationManagerSource = new ExecutorAllocationManagerSource
```

Comment:
ah, never mind it's used in SparkContext, then it really shouldn't be a field in this class. Let's fix that separately